api/internal/core/entity/format.go

```diff
@@ -17,26 +17,30 @@
 package entity
 
 import (
-	"log"
 	"strconv"
 	"strings"
+
+	"github.com/apisix/manager-api/log"
 )
 
 func NodesFormat(obj interface{}) interface{} {
 	var nodes []*Node
-	if value, ok := obj.(map[string]float64); ok {
+	switch objType := obj.(type) {
+	case map[string]float64:
+		log.Infof("nodes type: %v", objType)
 		var strArr []string
+		value := obj.(map[string]float64)
 		for key, val := range value {
 			node := &Node{}
 			strArr = strings.Split(key, ":")
```

Comment:
The key splitting logic can be replaced by using `net.SplitHostPort`, we may encapsulate it with port validation and IP/domain check.

Reply:
updated.